src/components/crewretrieval.tsx

```diff
@@ -51,6 +52,10 @@ const rarityOptions = [
 	{ key: '5', value: '5', text: '5' }
 ];
 
+const collectionsOptions = [
+	{ key: null, value: null, text: 'None' }
```

Comment:
Should the text here be 'Any' rather than 'None'?  One interpretation of None, would be crew not in any collections, whereas this effectively clearing the collections filter.

Reply:
Good catch!

I merely copied that from `const rarityOptions = []`, which should probably receive the same change, if we want to be consistent.

Reply:
Rarity is less of an issue, as the box does say "minimum rarity", but yes, might as well change them both to be the same.  Stick another commit on this with both of those changed and I'll merge it